deploy/dashboards/grafana-dashboard-provisioning.configmap.yaml

```diff
@@ -599,7 +599,7 @@ data:
                 "type": "prometheus",
                 "uid": "PDD8BE47D10408F45"
               },
-              "expr": "sum(rate(provisioning_source_availability_check_request_total{error=\"false\"}[$__range]))/sum(rate(provisioning_source_availability_check_request_total[$__range]))",
+              "expr": "sum(rate(provisioning_source_availability_check_request_total{exported_service="provisioning"}[$__range]))/sum(rate(provisioning_http_request_total{exported_service="provisioning", path="/api/provisioning/v1/availability_status/sources/"}[$__range]))",
```

Comment:
The service label does not need to be used anywhere. The metric name itself is unique I believe. A nitpick.